zerver/tests/test_import_export.py

```diff
@@ -296,6 +297,58 @@ def _setup_export_files(self, realm: Realm) -> Tuple[str, str, str, bytes]:
+            'INFO:root:Successfully imported <class \'zerver.models.UserHotspot\'> from zerver_userhotspot.',
+            'INFO:root:Successfully imported <class \'zerver.models.MutedTopic\'> from zerver_mutedtopic.',
+            'INFO:root:Successfully imported <class \'zerver.models.Service\'> from zerver_service.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserGroup\'> from zerver_usergroup.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserGroupMembership\'> from zerver_usergroupmembership.',
+            'INFO:root:Successfully imported <class \'zerver.models.BotStorageData\'> from zerver_botstoragedata.',
+            'INFO:root:Successfully imported <class \'zerver.models.BotConfigData\'> from zerver_botconfigdata.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserPresence\'> from zerver_userpresence.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserActivity\'> from zerver_useractivity.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserActivityInterval\'> from zerver_useractivityinterval.',
+            'INFO:root:Successfully imported <class \'zerver.models.CustomProfileField\'> from zerver_customprofilefield.',
+            'INFO:root:Successfully imported <class \'zerver.models.CustomProfileFieldValue\'> from zerver_customprofilefieldvalue.',
+            'INFO:root:Importing avatars',
+            'INFO:root:Importing uploaded files',
+            'INFO:root:Importing emojis',
+            'INFO:root:Importing realm icons and logos',
+            'INFO:root:Successfully rendered Markdown for message batch',
+            'INFO:root:Successfully imported <class \'zerver.models.Message\'> from zerver_message.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserMessage\'> from zerver_usermessage[1].',
+            'INFO:root:Successfully imported <class \'zerver.models.Reaction\'> from zerver_reaction.',
+            'INFO:root:Successfully imported <class \'zerver.models.Attachment\'> from zerver_attachment.',
+            'INFO:root:Successfully imported M2M table zerver_attachment_messages',
+            'INFO:root:Successfully imported <class \'analytics.models.RealmCount\'> from analytics_realmcount.',
+            'INFO:root:Successfully imported <class \'analytics.models.UserCount\'> from analytics_usercount.',
+            'INFO:root:Successfully imported <class \'analytics.models.StreamCount\'> from analytics_streamcount.'
+        ])
+        self.assertRegexpMatches(logs[31], r'^INFO:root:Importing message dump .*/test-export/messages-000001.json$')
+        self.assertRegexpMatches(logs[36], r'^INFO:root:Importing attachment data from .*/test-export/attachment.json$')
+        self.assertRegexpMatches(logs[39], r'^INFO:root:Importing analytics data from .*/test-export/analytics.json$')
+
```

Comment:
I think the new `check_logs` function may be more complexity than it's worth, since it'll make various changes to the import/export codebase more fragile; @hackerkid @showell @alexmv thoughts?  

For now I'll merge the other pieces without this file.

Reply:
I agree. I don't think it makes much sense to do a check like this. We already have checks for the actual functionalities.

Reply:
> We already have checks for the actual functionalities.

So should I just use `self.assertLogs()` with these functions which will suppress these logs while running tests and not assert them? Earlier also(with logging mocks) logs were only suppressed and were not asserted.

Reply:
@m-e-l-u-h-a-n Yes. I think you don't have to verify the content of these logs. You just have to suppress them